src/app/campaign/store/models/flight.models.ts

```diff
@@ -6,6 +6,7 @@ export interface FlightZone {
   id: string;
   label?: string;
   url?: string;
+  pingbacks?: string[];
```

Comment:
If the pingbacks can be an empty array instead of null, I put in a one line fix for this on the setup of the flight zones form group to use empty array when pingbacks are null/not set.

Reply:
Fixed!